CMakeLists.txt

```diff
@@ -734,11 +734,6 @@ if(WITH_OPTIM)
         if(WITH_SSE41)
             check_sse41_intrinsics()
             if(HAVE_SSE41_INTRIN)
-                add_definitions(-DX86_SSE41 -DX86_SSE41_ADLER32)
```

Comment:
We could leave `-DX86_SSE41` as placeholder, so the block isn't completely empty. Same for configure, even though it didn't have it before... Other option would be to negate the test so only the line `set(WITH_SSE41 OFF)` would stay...